src/test/java/com/google/firebase/database/MapperTest.java

```diff
@@ -248,6 +252,31 @@ public void primitiveDeserializeLong() {
     } catch (DatabaseException e) { // ignore
     }
   }
+  
+  @Test
+  public void bigIntegerDeserialize() {
+    BigIntegerBean beanBigInteger = deserialize("{'value':'1'}", BigIntegerBean.class);
+    assertEquals(0,BigInteger.ONE.compareTo(beanBigInteger.getValue()));
+    
+    BigIntegerBean beanBigIntegerBig =
```

Comment:
Can you use a value that is guaranteed to be larger then what can fit in a `long`? Something like:

```
BigInteger value = new BigInteger(String.valueOf(Long.MAX_VALUE)).add(BigInteger.ONE);
```